src/platform/utilities/tests/ui/index.unit.spec.jsx

```diff
@@ -1,44 +1,119 @@
+
+    it('should format SSN properly with dashes when fewer than 6 digits are entered', () => {
+      const result = formatSSN('1234');
+      expect(result).to.equal('123-4');
+    });
+
+    it('should format SSN properly with dashes when between 7 and 9 digits are entered', () => {
+      const result = formatSSN('1234567');
+      expect(result).to.equal('123-45-67');
+    });
   });
-  it('should return true for class component', () => {
-    class MyComponent extends React.Component {
-      constructor() {
-        super();
-        this.state = { test: true };
-      }
 
-      render() {
-        return <div>{this.state.test}</div>;
+  describe('isReactComponent', () => {
+    it('should return true for function component', () => {
+      expect(isReactComponent(() => <div />)).to.be.true;
+    });
+    it('should return true for class component', () => {
+      class MyComponent extends React.Component {
+        constructor() {
+          super();
+          this.state = { test: true };
+        }
+
+        render() {
+          return <div>{this.state.test}</div>;
+        }
       }
-    }
-    expect(isReactComponent(MyComponent)).to.be.true;
-  });
-  it('should return true for memoized component', () => {
-    expect(isReactComponent(React.memo(() => <div />))).to.be.true;
-  });
-  it('should return false for regular element', () => {
-    expect(isReactComponent(<div />)).to.be.false;
+      expect(isReactComponent(MyComponent)).to.be.true;
+    });
+    it('should return true for memoized component', () => {
+      expect(isReactComponent(React.memo(() => <div />))).to.be.true;
+    });
+    it('should return false for regular element', () => {
+      expect(isReactComponent(<div />)).to.be.false;
+    });
+    it('should return false for fragment', () => {
+      expect(isReactComponent(<>Test</>)).to.be.false;
+    });
+    it('should return false for string', () => {
+      expect(isReactComponent('Test')).to.be.false;
+    });
+    it('should return false for number', () => {
+      expect(isReactComponent(3)).to.be.false;
+    });
+    it('should return false for null', () => {
+      expect(isReactComponent(null)).to.be.false;
+    });
+    it('should return false for undefined', () => {
+      expect(isReactComponent()).to.be.false;
+    });
   });
-  it('should return false for fragment', () => {
-    expect(isReactComponent(<>Test</>)).to.be.false;
-  });
-  it('should return false for string', () => {
-    expect(isReactComponent('Test')).to.be.false;
-  });
-  it('should return false for number', () => {
-    expect(isReactComponent(3)).to.be.false;
-  });
-  it('should return false for null', () => {
-    expect(isReactComponent(null)).to.be.false;
-  });
-  it('should return false for undefined', () => {
-    expect(isReactComponent()).to.be.false;
+
+  // Not testing the scroll part of this function
+  describe('customScrollAndFocus', () => {
+    it('should focus om h3 when no param is passed', () => {
```

Comment:
Noticed typo:
    it('should focus **om** h3 when no param is passed', () => {
correcting to "on"